koans/src/java7/AboutStringsInSwitch.java

```diff
@@ -0,0 +1,27 @@
+package java7;
+
+import com.sandwich.koan.Koan;
+
+import static com.sandwich.koan.constant.KoanConstants.__;
+import static com.sandwich.util.Assert.assertEquals;
+
+public class AboutStringsInSwitch {
+
+    @Koan
+    public void stringsInSwitchStatement () {
+        String[] animals = {"Dog", "Cat", "Tiger", "Elephant", "Zebra"};
+        String dangerous=null;
+        String notDangerous=null;
+        for (String animal : animals){
+            switch (animal) {
+                case "Tiger":
+                        dangerous = animal;
```

Comment:
I wonder if there should be a `continue` statement in this `case` block.  The result will still be the same (the `dangerous` variable will have the value `"Tiger"`), but I wonder if that would make the koan more readable (and remind the student about falling through to the next `case` statement).